### ISSUE TITLE
[Accessibility]: Accordion - focus indicator overlap

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/vets-design-system-documentation#5368]
### Point of contact

Jamie Klenetsky Fay

### How severe do you think this issue is?

Low

### Severity rationale

Focus indicator is visible and buttons are selectable, just cut off at the bottom.

### URL where issue occurs

https://design.va.gov/storybook/iframe.html?globals=&args=&id=uswds-va-accordion--default&viewMode=story

### Component or pattern (if applicable)

va-accordion

### Environment details

- OS: Mac
- Browser: Chrome

### Steps to reproduce

1. Tab to the "Expand all" button - note the focus indicator "disappearing" beneath the accordion header button
2. Tab to the "Collapse all" button - same thing

It's more pronounced if you increase the browser zoom to 300%.

### Observed behavior

The focus indicator is overlapped by the accordion header button.

### Expected behavior / Acceptance criteria

The focus indicator should be fully visible.

### Supporting evidence (screenshots, recordings, code snippets)

Examples at different zoom levels:

<img width="335" height="88" alt="Image" src="https://github.com/user-attachments/assets/4a0660bb-de50-477c-ab28-f855b1dd4327" />

<img width="617" height="183" alt="Image" src="https://github.com/user-attachments/assets/220be4d2-484b-419c-8aed-b393fedd0cca" />

### Type of issue

- [ ] Axe-core
- [ ] Cognitive | user flows & navigating
- [x] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Content style guide
- [x] Focus
- [ ] Headings
- [ ] Keyboard
- [ ] Link and button labels
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Screenreader | assistive tech & device support
- [ ] Semantic HTML
- [ ] URL standards
- [ ] Zoom or Magnification
- [ ] Something else
